src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/AddDefendantLitigationFriendCallbackHandler.java

```diff
@@ -50,9 +62,9 @@
     private CallbackResponse aboutToSubmit(CallbackParams callbackParams) {
         CaseData caseDataUpdated = callbackParams.getCaseData().toBuilder()
             .businessProcess(BusinessProcess.ready(ADD_DEFENDANT_LITIGATION_FRIEND))
-            .respondent1LitigationFriendDate(LocalDateTime.now())
-            .respondent1LitigationFriendCreatedDate(
-                ofNullable(callbackParams.getCaseData().getRespondent1LitigationFriendCreatedDate())
+            .genericLitigationFriendDate(LocalDateTime.now())
```

Comment:
As we are now using the `genericLitigationFriend` field in the event + multiparty is disabled. We should extract the contents of `genericLitigationFriend` and populate these into the `respondent1LitigationFriend`
and also need to set the Date and Created Date as it was done previously.

With this implementation we are effectively doing a hard swap on the event. 
The field the user populates in the event will be the generic field. Which we will programatically copy to respondent1 + add the dates. Happy to have a call if that doesnt make any sense!